photutils/isophote/tests/test_fitter.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import pytest
+
+import numpy as np
+from astropy.io import fits
+
+from photutils.isophote import build_test_data
+from photutils.isophote.build_test_data import DEFAULT_POS
+from photutils.isophote.geometry import Geometry, DEFAULT_EPS
+from photutils.isophote.integrator import MEAN, MEDIAN
+from photutils.isophote.harmonics import fit_1st_and_2nd_harmonics
+from photutils.isophote.sample import Sample, CentralSample
+from photutils.isophote.isophote import Isophote
+from photutils.isophote.fitter import Fitter, CentralFitter
+from photutils.isophote.tests.test_data import TEST_DATA
+
+
+def test_gradient():
+
+    test_data = build_test_data.build()
+
+    sample = Sample(test_data, 40.)
+    sample.update()
+
+    assert sample.mean == pytest.approx(200.02, abs=0.01)
```

Comment:
These are all failing at the moment, as we are still using the astropy bundled ancient version of pytest.

@larrybradley - what do you think, shall we aim to merge #491 asap or rather work around to still support pytest 2.x?